_posts/2024-05-23-get-involved-with-pyos.md

```diff
@@ -0,0 +1,90 @@
+
+Please feel free to use any information from [our website](https://www.pyopensci.org/) and our [pyOpenSci lightning talk template [currently WIP]](https://docs.google.com/presentation/d/1205l8y75SI9spdklKnxDZzwwjArHvpeIpwFGZOvVLjc/edit?usp=sharing), to share about the work pyOpenSci is doing. And don’t forget to share your personal experiences with pyOpenSci as well!
+
+## <i class="fa-solid fa-handshake-angle"></i> Volunteer with pyOpenSci
+If you're looking to be more hands-on with pyOpenSci, consider joining our Review Team or Editorial Board, digging into some code and working from our Help Wanted board, or writing a guest blog post!
+
+### pyOpenSci's Review Team
+pyOpenSci welcomes reviewers from a diversity of background and with varying levels of Python technical expertise. Some of the basic things that we look for in a reviewer include:
+
+* Familiarity with using the Python programming language.
+* Ability to evaluate a Python package for usability and documentation quality.
+* Ability to provide a technical review of Python package structure and code quality/approach to solving the problems that the package seeks to address.
+
+We like to have a mix of technical and usability focus in our reviews so it’s ok if you don’t have all of the above skills! And if you are interested in peer review but have never reviewed before, we offer a mentorship program where we will pair you up with someone who has more experience reviewing code. From this experience you can learn more and empower yourself with code review skills. Software review skills are generally useful in data science, so they are skills worth investing in!
+
+To join our review team, [fill out our reviewer form](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform). We will contact you if we have a package that we need reviewers for. It’s OK if you’ve never reviewed a package before! We’ll walk you through it.
+
+### pyOpenSci's Editorial Team
+The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
+
+* Finding reviewers from diverse backgrounds who have a mixture of scientific domain and Python experience.
+* Overseeing the entire review process for a package ensuring it runs in a timely and efficient manner.
+* Supporting the submitting authors and reviewers in answering questions related to the review.
+* Determining whether that package should be accepted into the pyOpenSci ecosystem once the review has wrapped up.
+
+It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. We do not expect any single editor to be an expert in all of these areas!
+
+### Apply to be a pyOpenSci editor
```

Comment:
question - this is great. i am not sure if this is a valid concern. we don't want people signing up all of the time... it's only right now that we need editors. what happens otherwise is peope llose interest and we waist time reaching out to them in say 3 months if you know what i mean? can we provide a time cap like through june or july or something?

Reply:
sure thing!